plenoptic/metric/perceptual_distance.py

```diff
@@ -70,11 +71,13 @@ def _ssim_parts(img1, img2, dynamic_range, pad=False):
 
     real_size = min(11, height, width)
     std = torch.tensor(1.5).to(img1.device)
-    window = circular_gaussian2d(real_size, std, n_channels)
+    window = circular_gaussian2d(real_size, std=std)
 
     # these two checks are guaranteed with our above bits, but if we add
     # ability for users to set own window, they'll be necessary
-    if (window.sum((-1, -2)) > 1).any():
+    window_sum = window.sum((-1, -2))
+    if not torch.allclose(window_sum, torch.ones_like(window_sum)):
+        print(window.sum((-1, -2)))
         warnings.warn("window should have sum of 1! normalizing...")
```

Comment:
This `print` should be included in the warning (`warnings` and `print` get separately caught if the user wants to filter by them)

Reply:
I think the `print` statement was used for debugging and left here by mistake. I'll just delete it.